components/Header/src/HeaderLogic.tsx

```diff
@@ -0,0 +1,247 @@
+import React, { useEffect, useRef, useState } from 'react';
+import HeaderLogo from '@gemeente-denhaag/header-logo';
+import ResponsiveContent from '@gemeente-denhaag/responsive-content';
+import Link from '@gemeente-denhaag/link';
+import { MenuButtonExpandable, MobileMenu } from '@gemeente-denhaag/menu';
+import './index.scss';
+import Header from './Header';
+import HeaderContent from './HeaderContent';
+import HeaderLogoContainer from './HeaderLogoContainer';
+import HeaderActions from './HeaderActions';
+import HeaderMobileActions from './HeaderMobileActions';
+import HeaderAction from './HeaderAction';
+import Breadcrumb from '@gemeente-denhaag/breadcrumb';
+import { Sheet, SheetOverlay, SheetContainer, SheetDialog, useEscapeKey, useScreenSize } from '@gemeente-denhaag/sheet';
+import IconButton from '@gemeente-denhaag/iconbutton';
+import { CloseIcon, ArrowRightIcon, LogOutIcon } from '@gemeente-denhaag/icons';
+import { LinkGroup, LinkGroupList, LinkGroupListItem } from '@gemeente-denhaag/link-group';
+import { LanguageSwitcherLogic, LanguageSwitcherLogicProps } from '@gemeente-denhaag/language-switcher';
+import { Heading4 } from '@gemeente-denhaag/typography';
+import { Button } from '@gemeente-denhaag/button';
+import './index.scss';
+
+export interface HeaderLogicProps {
+  breadcrumbs: Array<LinkItemProps>;
+  userprofileMenu: MenuProps;
+  languageSwitcherMenu: LanguageSwitcherLogicProps;
+  mobileMenu: Array<NavigationGroupProps>;
+}
+
+interface LinkItemProps {
+  label: string;
+  url: string;
+}
+
+interface MenuProps {
+  label: string;
+  navigationGroups: Array<NavigationGroupProps>;
+}
+
+interface NavigationGroupProps {
+  label: string;
+  url?: string;
+  navigation?: Array<LinkItemProps>;
+}
+
+const NavigationGroup: React.FC<NavigationGroupProps> = (props: NavigationGroupProps) => {
+  const navigationLinks = props.navigation?.map((navigationLink, key) => {
+    return (
+      <LinkGroupListItem key={key}>
+        <Link icon={<ArrowRightIcon />} iconAlign="start" href={navigationLink.url}>
+          {navigationLink.label}
+        </Link>
+      </LinkGroupListItem>
+    );
+  });
+
+  return (
+    <LinkGroup>
+      <Heading4>{props.label}</Heading4>
+      <LinkGroupList>{navigationLinks}</LinkGroupList>
+    </LinkGroup>
+  );
+};
+
+export const HeaderLogic: React.FC<HeaderLogicProps> = (props: HeaderLogicProps) => {
+  const menuConstants = {
```

Comment:
Might be nice to get the breakpoint from the props instead? Maybe even with the 1024 as the default? 

`{mobileBreakpoint = 1024, ...props}: HeaderLogicProps`

Reply:
Done!